pom.xml

```diff
@@ -384,7 +391,7 @@
             <configuration>
               <failOnWarning>true</failOnWarning>
               <ignoredDependencies>
-                <ignoredDependency>log4j:log4j:jar:*</ignoredDependency>
+                <ignoredDependency>org.apache.logging.log4j:*</ignoredDependency>
                 <ignoredDependency>org.slf4j:slf4j-log4j12:jar:${slf4j.version}</ignoredDependency>
```

Comment:
slf4j-log4j12 shouldn't be on the class path at all. That was the log4j 1.2 runtime binding for slf4j 1.7. Should probably remove that here. If that causes a failure, it should be excluded as a transitive dependency from any dependency that's bringing it in.